client/chat/channel.tsx

```diff
@@ -171,7 +167,7 @@ const UserListEntry = React.memo<UserListEntryProps>(props => {
   )
 })
 
-const UsersVirtualizedList = styled(VirtualizedList)`
+const UsersVirtualizedList = styled(VariableSizeList)`
```

Comment:
These styles were to override something that react-virtualized set, shouldn't be necessary with the new library.